### DISCUSSION TITLE
Welcome to SillyTavern Discussions! (English only)

### DISCUSSION THREAD

**Discussion by Cohee1207** [SillyTavern/SillyTavern#2934] [Announcements]
## 👋 Welcome!

Use Discussions to ask non-technical questions that do not relate to bugs or feature requests.

**Comment by 1550542423mzc** [comment]
Microsoft Windows [版本 10.0.22631.4317]
(c) Microsoft Corporation。保留所有权利。

C:\MySpecialFolder>git clone https://github.com/SillyTavern/SillyTavern -b release
fatal: could not create work tree dir 'SillyTavern': Permission denied

C:\MySpecialFolder>
How can I do this?
